packages/serverless-logic-web-tools/src/editor/hooks/useDeployDropdownItems.tsx

```diff
@@ -201,13 +201,31 @@ export function useDeployDropdownItems(props: Props) {
     [devMode.isEnabled, props.workspaceFile.name]
   );
 
+  const onDeployWarningAlert = useGlobalAlert(
```

Comment:
```suggestion
  const validationWarningAlert = useGlobalAlert(
```
I would remove "deploy" from the name so we can use it for the dev mode uploading also.